### DISCUSSION TITLE
ns apple-login email  password

### DISCUSSION THREAD

**Discussion by abdallahkadour** [NativeScript/NativeScript#9898] [General]
This simple command prints the following on the console: 
`Request failed with status code 400`
I am pretty sure that my credentials are correct, also I am entering the 2-factor code correctly. Any Idea? 
Also Why only the status code is printed why not the error response?
Thanks in advance

**Comment by rigor789** [comment]
Double check your credentials, and make sure you are using the latest NativeScript CLI (npm i -g nativescript). Everything seems to work on my end - just tried logging in, and it works.
